bot-utilites.php

```diff
@@ -11,3 +11,8 @@
 if ( ! defined( 'WPINC' ) ) {
     die;
 }
+
```

Comment:
`includes/config.php` needs to be added as an include here.  Alternatively it may make more sense to just define those constants in this main plugin file to reduce the number of files being processed when the plugin is loaded.

Reply:
Other note: the filename for this file is misspelled (`bot-utilites.php`)